commercetools.Sdk/Tests/commercetools.Api.Serialization.Tests/AttributesTests.cs

```diff
@@ -229,5 +235,21 @@ public void IntDeserialization()
             Assert.IsType<long>(attributes.Get("t").Value);
             Assert.Equal("{\"id\":0,\"attributes\":[{\"name\":\"t\",\"value\":13}]}", serializerService.Serialize(deserialized));
         }
+
+        [Fact]
+        public void DecimalSerialization()
+        {
+            var serializerService = this._serializationFixture.SerializerService;
+            var attributes = new List<Attribute>
+                {
+                    new DecimalAttribute() { Name = "decimal", Value = 13m },
+                    new DecimalAttribute() { Name = "decimal-zero", Value = 13.0m },
+                    new DecimalAttribute() { Name = "double", Value = 13d },
+                    new DecimalAttribute() { Name = "double-zero", Value = 13.0d },
+                    new LongAttribute() { Name = "long", Value = 13 }
+                };
+            Assert.Equal("[{\"name\":\"decimal\",\"value\":13},{\"name\":\"decimal-zero\",\"value\":13.0},{\"name\":\"double\",\"value\":13},{\"name\":\"double-zero\",\"value\":13},{\"name\":\"long\",\"value\":13}]", serializerService.Serialize(attributes));
```

Comment:
Should we have `13.0` for `double-zero`?

Reply:
Btw a real fix to preserve or enforce a format for the decimal values can only be done with .NET 6.0 as before the Utf8JsonWriter doesn't have the ability to write a raw or preformatted value to Json

Reply:
Maybe it makes more sense to align with what is used in entire system (Merchant Center, Import SDK) and have only one attribute class for Number attribute type: `NumberAttribute`?

Reply:
For backwards compatibility reasons I decided to go with a configuration option which disables the long/decimal detection. Still it will then be possible to cast LongAttribute to DecimalAttribute and vice versa.

Btw the ImportApi is in this regard a little bit different as it has a polymorphic model for attributes versus the union type in the Api.